h2o-py/h2o/estimators/deeplearning.py

```diff
@@ -36,20 +36,20 @@ class H2ODeepLearningEstimator(H2OEstimator):
     def __init__(self, **kwargs):
```

Comment:
all changes in estimators below are generated code... not sure why those are stored in git

Reply:
Thanks for the heads-up